src/main/java/org/isf/medicalstockward/manager/MovWardBrowserManager.java

```diff
@@ -224,7 +224,40 @@ public List<MedicalWardForPrint> convertWardDrugs(Ward wardSelected, List<Medica
 		Collections.sort(drugPrint);
 		return drugPrint;
 	}
-
+	
+	/**
+	 * Get the  MedicalWard for the specified criteria.
+	 *
+	 * @param wardCode - the ward id  
+	 * @param medical - the mecical id 
+	 * @param lot - the lot id
+	 * @return the retrieved medical.
+	 * @throws OHServiceException if an error occurs retrieving the medical.
+	 */
+	public MedicalWard getMedicalWardByWardAndMedical(String wardCode, int medical, String lotCode) throws OHServiceException {
+		return ioOperations.getMedicalWardByWardAndMedical(wardCode, medical, lotCode);
+	}
+	
+	/**
+	 * Deletes the  MedicalWard.
+	 *
+	 * @param medWard - the MedicalWard to delete. 
+	 * @throws OHServiceException.
+	 */
+	public void deleteMedicalWard(MedicalWard medWard) throws OHServiceException {
+		ioOperations.deleteMedicalWard(medWard);
+	}
+	
+	/**
+	 * Updates the  MedicalWard.
+	 *
+	 * @param MedicalWard - the MedicalWard to update. 
```

Comment:
```suggestion
	 * @param medWard - the MedicalWard to update. 
```
Use the correct parameter name.   And why isn't there a `{@link }` to the object?